packages/web3-eth-contract/src/contract.ts

```diff
@@ -373,7 +374,8 @@ export class Contract<Abi extends ContractAbi>
 			gas: options?.gas ?? options?.gasLimit,
 			gasPrice: options?.gasPrice,
 			from: options?.from,
-			input: options?.input ?? options?.data,
+			input: options?.input,
```

Comment:
@luu-alex  so this will use same logic at contract level. like [we have here](https://github.com/web3/web3.js/issues/6183#issuecomment-1654656753)?
as wt ever user will specify ( either `input` or `data`) that will be sent to node.

Reply:
Yes and if theres no specification, it will default to input which is the behavior right now.